delivery_schenker_picking_volume/__manifest__.py

```diff
@@ -11,5 +11,5 @@
     "author": "MT Software, Odoo Community Association (OCA)",
     "license": "AGPL-3",
     "depends": ["delivery_schenker", "stock_picking_volume"],
-    "auto_install": True,
+    "auto_install": False,
```

Comment:
I don't get why this change since the refactoring where you don't use anymore the picking volume

Reply:
Because i am using the method _get_volume_for_qty instead of picking.volume. 
I would not install the module automatically, that should always be an explicit choice.

Reply:
But `_get_volume_for_qty` in `stock_picking_volume` is returning the same as the base module here https://github.com/OCA/delivery-carrier/pull/675/files#diff-f7e2841d64dbfbb11160cdc9a3518621837855957cbfc81446649cb15755f37eR346

So nothing is changing and auto_install can be True in my opinion. What is optional is to install https://github.com/OCA/stock-logistics-warehouse/tree/14.0/stock_picking_volume_packaging that improves the computation and could lead to a different result

Reply:
Ah yes you are right.